spec/requests/organisations_controller_spec.rb

```diff
@@ -717,23 +717,29 @@
 
         context "when viewing a specific organisation's lettings logs" do
           let(:number_of_org1_lettings_logs) { 2 }
+          let(:number_of_org1_managed_lettings_logs) { 2 }
           let(:number_of_org2_lettings_logs) { 4 }
 
           before do
             create_list(:lettings_log, number_of_org1_lettings_logs, created_by: user)
+            create_list(:lettings_log, number_of_org1_managed_lettings_logs, managing_organisation: user.organisation)
             create(:lettings_log, created_by: user, status: "pending", skip_update_status: true)
             create_list(:lettings_log, number_of_org2_lettings_logs, created_by: nil, owning_organisation_id: unauthorised_organisation.id, managing_organisation_id: unauthorised_organisation.id)
 
             get "/organisations/#{organisation.id}/lettings-logs", headers:, params: {}
           end
 
           it "only shows logs for that organisation" do
-            expect(page).to have_content("#{number_of_org1_lettings_logs} total logs")
+            expect(page).to have_content("#{number_of_org1_lettings_logs + number_of_org1_managed_lettings_logs} total logs")
```

Comment:
Can we also make a log that is owned and managed by the org and ensure it isn't included twice in the count (or the page)

Reply:
Sure!